internal/saucecloud/cloud.go

```diff
@@ -259,3 +259,18 @@ func (r *CloudRunner) validateTunnel(id string) error {
 	log.Info().Msg("Tunnel is ready!")
 	return nil
 }
+
+func (r *CloudRunner) dryRun(project interface{}, files []string, sauceIgnoreFile string, suiteNames string) error {
+	tmpDir, err := ioutil.TempDir("./", "sauce-app-payload-*")
+	if err != nil {
+		return err
+	}
+	log.Info().Msgf("Detected following test suites: %s.", suiteNames)
```

Comment:
Shows the intent a bit more:
```suggestion
	log.Info().Msgf("The following test suites would have run: [%s].", suiteNames)
```